src/main/java/com/stripe/model/Charge.java

```diff
@@ -79,19 +78,53 @@ public void setAmountRefunded(Long amountRefunded) {
 	}
 
 	public String getApplicationFee() {
-		return applicationFee;
+		if (this.applicationFee==null) {
+			return null;
+		}
+		return this.applicationFee.getId();
+	}
+
+	public void setApplicationFee(String applicationFeeID) {
+		if (this.applicationFee==null || (this.applicationFee.isExpanded() && (this.applicationFee.getExpanded().getId() != applicationFeeID))) {
+			this.applicationFee = new ExpandableField<ApplicationFee>(applicationFeeID, null);
+		}
+		this.applicationFee.setId(applicationFeeID);
+	}
+
+	public ApplicationFee getApplicationFeeObject() {
+		if (this.applicationFee==null) {
+			return null;
+		}
+		return this.applicationFee.getExpanded();
 	}
 
-	public void setApplicationFee(String applicationFee) {
-		this.applicationFee = applicationFee;
+	public void setApplicationFeeObject(ApplicationFee c) {
```

Comment:
@brandur Since expanding is optional, new fields should have both right? You might still want to get the review id stored in the `review` property instead of expanding it and would ask for `getReview()` in that case no?